pages/common/sd.md

```diff
@@ -0,0 +1,19 @@
+# sd
+
+> `sd` is an intuitive find & replace CLI.
+
+- Trim some whitespace using regex:
+
+`echo 'lorem ipsum 23   ' | sd '\s+$' ''`
```

Comment:
We usually add tokens around custom input examples.
```suggestion
`{{echo 'lorem ipsum 23   '}} | sd '\s+$' ''`
```

Reply:
The changes look good, but just for my own sake, I'm unfamiliar with the double-curly brace. I've used singles before in this context. I tried the updated diff in both zsh and bash, and neither work for me. Am I missing something obvious? 😅

Reply:
It's actually the token syntax for the `tldr` project. 👍 It allows the `tldr` clients to highlight customisable entries in the pages. The [style guide](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/style-guide.md#token-syntax) explains how this works.

![](https://image.prntscr.com/image/dSbxR_5STquWaW9zJUbutg.png)

Reply:
🤦🏼‍♂️

Thank you.